### PR TITLE
Show only hints if numeric value is gt 0

### PR DESCRIPTION
**Pull request by JanAckermann**
Fixes: #344

### CODE REVIEW

**Comment by phil-davis** [comment]
commit message is the wrong way around? "lt" should be "gt".
It might be worth fixing that, to avoid confusion for other devs in future.

**Comment by JanAckermann** [comment]
@phil-davis thanks, holy monday !
Also adjusted min vals

File: templates/admin.php [hunk 1]
```diff
@@ -38,31 +38,31 @@
...
 					<input type="checkbox" name="spv_lowercase_checked" <?php if ($_['spv_lowercase_checked']): ?> checked="checked"<?php endif; ?>>
-					<input type="number" name="spv_lowercase_value" min="0" max="255" value="<?php p($_['spv_lowercase_value']) ?>"> <?php p($l->t('lowercase letters'));?>
+					<input type="number" name="spv_lowercase_value" min="1" max="255" value="<?php p($_['spv_lowercase_value']) ?>"> <?php p($l->t('lowercase letters'));?>
 				</label>
 			</li>
 			<li>
 				<label>
 					<input type="checkbox" name="spv_uppercase_checked" <?php if ($_['spv_uppercase_checked']): ?> checked="checked"<?php endif; ?>>
-					<input type="number" name="spv_uppercase_value" min="0" max="255" value="<?php p($_['spv_uppercase_value']) ?>"> <?php p($l->t('uppercase letters'));?>
+					<input type="number" name="spv_uppercase_value" min="1 max="255" value="<?php p($_['spv_uppercase_value']) ?>"> <?php p($l->t('uppercase letters'));?>
```
Comment (jvillafanez) [inline templates/admin.php:53]: there is a missing double-quote after the "1"

**Comment by jvillafanez** [comment]
Just to confirm, the expectation is that the value is removed in order not to use the policy, I guess. I mean, if don't require uppercase chars, the value should be removed, not just set to 0. I trust it works well with the "number" input type.

**Review by jvillafanez — APPROVED** [review]

**Comment by JanAckermann** [comment]
@jvillafanez why this should be removed? From UX POV this makes no sense. I can deactivate a setting, and enable it later without the previous values getting lost.

image

**Comment by jvillafanez** [comment]
ok, got it. Didn't see the checkboxes. :+1:

**Pull request merged**